ui/ducks/ens.js

```diff
@@ -177,7 +180,13 @@ export function lookupEnsName(ensName) {
       let address;
       let error;
       try {
-        address = await provider.resolveName(trimmedEnsName);
+        // the writable property on the 'provider' object on the 'web3Provider' flips to false when stale
```

Comment:
Biggest nit of all time but if we're going to capitalize `This` on the next line, maybe we capitalize `The` here?